src/objects/zcl_abapgit_object_doma.clas.abap

```diff
@@ -391,6 +392,11 @@ CLASS ZCL_ABAPGIT_OBJECT_DOMA IMPLEMENTATION.
       valpos ASCENDING
       ddlanguage ASCENDING.
 
+    LOOP AT lt_dd07v ASSIGNING <ls_dd07v>.
+      CLEAR <ls_dd07v>-domname.
+      CLEAR <ls_dd07v>-valpos.
```

Comment:
`valpos` is used during deserialize (matching texts, sorting). we should not remove it.

Reply:
can it be set during deserialize? I guess its just `sy-index`?

Reply:
it's already doing that 
https://github.com/abapGit/abapGit/blob/main/src/objects/zcl_abapgit_object_doma.clas.abap#L268

all 'm saying is that we tried clearing it before and there were issues.

Reply:
I thought this was okay to clear as it is filled with sy-index before it is used, so the exported value should not even matter. But as originally I wanted only domname, i will undo this part if it's uncertain.